packages/common/src/lib/format-questionnaire-details.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * @typedef {import("../client/appeals-api-client").AppealCaseWithAppellant} AppealCaseWithAppellant
+ */
+
+/**
+ * @param {AppealCaseWithAppellant} caseData
+ * @param {keyof AppealCaseWithAppellant} propertyName
+ */
+exports.formatYesOrNo = (caseData, propertyName) => (caseData[propertyName] ? 'Yes' : 'No');
+
+// TODO the associated changed and affected listed building numbers may be more than 1
+// the current data model does not support multiple entries.
+// function will need refactoring once data model updated
+// /**
+//  * @param {AppealCaseWithAppellant} caseData
+//  */
+// exports.formatListedBuildings = (caseData) => {
+// 	const allListedBuildings = [];
+// 	const changedListedBuildingNumber = caseData.changedListedBuildingNumber;
+// 	const affectedListedBuildingNumber = caseData.affectedListedBuildingNumber;
+// 	const listedBuildingUrl = 'https://historicengland.org.uk/listing/the-list/list-entry/';
+
+// 	if (!changedListedBuildingNumber && !affectedListedBuildingNumber) {
+// 		return '';
+// 	}
+
+// 	if (changedListedBuildingNumber) {
+// 		allListedBuildings.push(
+// 			`<a href="${listedBuildingUrl}${changedListedBuildingNumber}">${changedListedBuildingNumber}</a>`
+// 		);
+// 	}
+
+// 	if (affectedListedBuildingNumber) {
+// 		allListedBuildings.push(
+// 			`<a href="${listedBuildingUrl}${affectedListedBuildingNumber}">${affectedListedBuildingNumber}</a>`
+// 		);
+// 	}
+
+// 	return allListedBuildings.length > 1
+// 		? allListedBuildings.join('<br>')
+// 		: allListedBuildings.toString();
+// };
+
+/**
+ * @param {AppealCaseWithAppellant} caseData
+ */
+exports.formatDesignations = (caseData) => {
+	if (caseData.designatedSites === 'None') {
+		return 'No';
+	}
+
+	if (caseData.designatedSites === 'other' && caseData.otherDesignationDetails) {
+		return `Other<br>${caseData.otherDesignationDetails}`;
+	}
+
+	if (caseData.designatedSites && caseData.designatedSites !== 'None') {
+		return caseData.designatedSites;
+	}
+
+	return '';
+};
+
+/**
+ * @param {AppealCaseWithAppellant} caseData
+ */
+exports.formatSensitiveArea = (caseData) =>
+	caseData.sensitiveArea ? `Yes<br>${caseData.sensitiveAreaDetails}` : 'No';
+
+/**
+ * @param {AppealCaseWithAppellant} caseData
+ */
+exports.formatEnvironmentalImpactSchedule = (caseData) => {
+	if (caseData.environmentalImpactSchedule === 'schedule-1') {
+		return 'Schedule 1';
+	}
+
+	if (caseData.environmentalImpactSchedule === 'schedule-2') {
+		return 'Schedule 2';
+	}
+
+	return 'No';
+};
+
+/**
+ * @param {AppealCaseWithAppellant} caseData
+ */
+exports.formatDevelopmentDescription = (caseData) => {
```

Comment:
A dictionary might be easier to read 

```
const developmentDescriptions = {
  'agriculture-aquaculture': 'Agriculture and aquaculture',
  'change-extensions': 'Changes and extensions',
  ...
}

exports.formatDevelopmentDescription = (caseData) => developmentDescriptions[caseData. developmentDescription] || 'None
```

Reply:
The function needed an explicit check on the `caseData.developmentDescription` as the IDE was complaining it could be undefined with the given example

Reply:
nice, lgtm